### ISSUE TITLE
thermal mode

### ISSUE THREAD

**Issue by unkown367** [studio-minus/ppg-bugs-and-requests#117]
description
it allows you to see the temp in the area

purpose
allows you to see the heat of a object and the ambient heat

**Comment by unkown367** [comment]
![thermal-scope (1)](https://user-images.githubusercontent.com/87345729/130331264-0d5f8449-e834-4229-8eaf-4f66c197a38f.jpg)
 like this

**Comment by JohnVcav** [comment]
that would be pretty useful

**Comment by roscosthebalm** [comment]
they should add the realistic part where bodys slowly get colder when a human dies

**Comment by mestiez** [comment]
> they should add the realistic part where bodys slowly get colder when a human dies

This already happens.

**Comment by LaCumbiaDelCoronavirus** [comment]
good idea but i consider that any thermal vision to actually do with constantly changing temperature may be very bad for fps.

**Comment by CUBE0R** [comment]
this is probably what the infrared thermometer is going to be, if i had to guess.
